build/webpack/_production.js

```diff
@@ -16,17 +16,37 @@ export default (webpackConfig) => {
   }
 
   debug('Apply ExtractTextPlugin to CSS loaders.')
+  let appliedExtractCSS = false
   webpackConfig.module.loaders = webpackConfig.module.loaders.map(loader => {
-    if (/css/.test(loader.test)) {
-      const [first, ...rest] = loader.loaders
+    if (!loader.loaders) {
+      return loader
+    }
+
+    let hasCSSLoader = false
+    loader.loaders.forEach(styleLoader => {
```

Comment:
Good point, but what you ever use `postcss` without `css-loader`? I only mention the `css` shorthand since I was actually planning on removing the `-loader` suffixes.

And as far as the inline function, I'm just trying to condense the logic to keep the configuration a bit smaller. Honestly not that big of a deal. I think it's nice to avoid flag variables as much as possible, so this was following along those lines.